### ISSUE TITLE
Waves under LinVST3

### ISSUE THREAD

**Issue by perlindgren** [osxmidi/LinVst3#8]
There seems to be a typo in the Waves.reg. 

The PR should fix that https://github.com/osxmidi/LinVst3/pull/7.

Overall the instructions to get Waves plugins working win LinVst is quite hard to follow. If possible please make clear (by example) how to get the dlls (if official sources exist, or else how you can obtain them from an existing WIN10 install), examples how to override, etc.

```
Waves Central v11 needs windows powershell which does not currently work.

Because Wine has some missing parts as compared to Windows (ie Robocopy, reg entries, some dll's etc) some things need to be installed and setup.

...
```
Does this mean that v11 is a no-go, and the text below just applies to waves 10 (or older)? (I assume so but it's not perfectly clear.)

```
1: Install Wine Staging (Ubuntu needs libfaudio0) and install winetricks cmd (to workaround possible cmd problems that can cause hangs).
```
Is that true/required also for the current 5.3 release?

```
2: Install the mfc42 and mfc42u 32bit dll overrides into ~/.wine/drive_c/windows/syswow64 (and optionally add the dll names in winecfg's Libraries tab)
```
Does that mean you should get `mfc42` and `mfc42u` from an existising WIN10 install? (In the dropdown under libraries, these are not found). When you manually add them (optionally), is it just the name(s) of the dlls or some additional parameters.

```
3: Install rktools.exe (Robocopy from Windows Server 2003 Resource Kit Tools)
```
In `wintricks` there is an option to install `psdk2003` (MS Platform SDK 2003), is that the same as the Resource Kit (with Robocopy). If not this maybe this link could be useful. https://www.microsoft.com/en-us/download/details.aspx?id=17657
However, when I tried running the latter (under wine 5.3) it asked for User Information, and pressing next seems to abort the program (at least it closes directly). According to the install instructions, files should be created under  %Program Files%\Windows Resource Kits\Tools folder, but nothing seems to have been installed.

```
4: Install the mfc140.dll override into ~/.wine/drive_c/windows/system32 (and optionally add the dll names in winecfg's Libraries tab)
```
Same as 2. 

```
5: Import Waves.reg (in LinVst's Waves folder) into regedit
```
Use the fixed one.

```
6: Install Waves Central
```
Which versions does apply? (10 and/or 11)

```
7: The vst 64 bit dll to wrap is WaveShellxxxxxx.dll in ~/.wine/drive_c/Program Files/VSTPlugIns/ (there might be more than one WaveShellxxxxxx.dll depending on additional Waves plugins installs)
```
Did not get that far.

```
7: A dialog box will popup (asking for a path) the first time a Waves V10 plugin is run, descend into the path by double clicking on (C:) and then double click on Program Files (x86) (or Program Files if Plug-Ins V10 can't be found) and then double click on Waves and then just highlight Plug-Ins V10 with a single click (don't descend into it) and then finish by clicking on open.
```
Seems to indicate the process works only for v10.

If one succeeds to install powershell under wine, would it be expected that v11 could could work with similar instructions as above?

**Comment by osxmidi** [comment]
There is no currently no way to get Powershell working under Wine so the current Waves Central won't work.

**Comment by niculw** [comment]
Waves 11 works under wine!!

https://github.com/PietJankbal/powershell-wrapper-for-wine/issues/1#issuecomment-704560804

Currently Waves does not open in a win10 prefix (thus v11 plugins dont work yet), so make sure the prefix is win7, win8 or win8.1.

When downloading the plugins. Check what plugins you have activated/can activate, and now download the v10 version of this, everything should be working. I have all my plugins working in reaper using linvst (so vst2) and im currently trying linvst3.
(Look at the readme for install instructions) You wanna download the 2 powershell files and then run "wine powershell" to set it up.
